commercetools/ui-kit
# Date inputs with months greater or equal to the 10th month do not highlight and submit.

fix issue where date inputs with months greater or equal to the 10th month do not highlight and submit.

**How can I test this?**
Reproducing the issue is a bit tricky. The fastest way to reproduce the issue is to comment out the `minValue` and `maxValue` props of the` <DateInput>` component in `packages/components/inputs/date-input/src/date-input.story.js`.
Go to date-input on your storybook, check to see that values with, for example, 12/12/2022 or any date with month >=10 (Oct, Nov and Dec) do not submit. Checkout to this current work branch to check again that this PR fixes it. 

Fixes: #2261
